FlowCrypt/src/main/java/com/flowcrypt/email/ui/activity/fragment/CreateMessageFragment.kt

```diff
@@ -1537,7 +1537,7 @@ class CreateMessageFragment : BaseFragment<FragmentCreateMessageBinding>(),
   }
 
   private fun isPasswordProtectedFunctionalityEnabled(): Boolean {
-    return accountViewModel.activeAccountLiveData.value?.useFES ?: return false
+    return accountViewModel.activeAccountLiveData.value?.useCustomerFesUrl ?: return false
   }
```

Comment:
You can remove this method now, we can enable password protected functionality for everybody.

Reply:
I would like to modify it via a separate issue. The current PR already has a lot of changes.